src/corelibs/U2Lang/src/support/SimpleWorkflowTask.cpp

```diff
@@ -148,6 +149,14 @@ SimpleMSAWorkflow4GObjectTask::SimpleMSAWorkflow4GObjectTask(const QString &task
     MultipleSequenceAlignment al = MSAUtils::createCopyWithIndexedRowNames(msaObjectPointer->getMultipleAlignment());
 
     MultipleSequenceAlignmentObject *msaObject = MultipleSequenceAlignmentImporter::createAlignment(msaObjectPointer->getEntityRef().dbiRef, al, os);
+    for (const MultipleSequenceAlignmentRow &row : al->getMsaRows()) {
+        if (row->getData().contains("*")) {
+            ioLog.info(tr("Symbols '*' has been replaced with 'X' during alignment."));
```

Comment:
I think, this sign's better to be shown just one time, not for every read with the `*` character. Like:

```
bool contains = false;
for(..) {
    if (...contains("*")) {
        contains = true;
    }
}
if (contains) {
    ioLog...
}
```